Octokit/Authentication/BearerTokenAuthenticator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Globalization;
+
+namespace Octokit.Internal
+{
+    class BearerTokenAuthenticator: IAuthenticationHandler
+    {
+        public void Authenticate(IRequest request, Credentials credentials)
+        {
+            Ensure.ArgumentNotNull(request, "request");
```

Comment:
for any `Ensure` checks can we please use `nameof(request)` rather than `"request"` ?

Reply:
Already did in some places. Will change that one too.